app/jobs/index_newly_published_vacancies_job.rb

```diff
@@ -0,0 +1,9 @@
+class IndexNewlyPublishedVacanciesJob < ApplicationJob
```

Comment:
Maybe a comment clarifying that this is for "scheduled" vacancies that become published as the current date started would help here.